src/Compilers/CSharp/Portable/Symbols/Source/SourceFieldLikeEventSymbol.cs

```diff
@@ -80,6 +80,11 @@ internal SourceFieldLikeEventSymbol(SourceMemberContainerTypeSymbol containingTy
                 // Don't initialize this.type - we'll just use the type of the field (which is lazy and handles var)
             }
 
+            if (inInterfaceType && !this.IsAbstract && !this.IsExtern)
+            {
+                diagnostics.Add(ErrorCode.ERR_EventNeedsBothAccessors, this.Locations[0], this);
```

Comment:
>Why is this an error?

Because non-abstract, non-extern events should declare both accessors with implementation.